ParabolInc/parabol
# feat: add default tasks to new team

# Description

resolved #6719

## Demo

https://user-images.githubusercontent.com/4997466/182640794-9a17f482-f8b6-4352-8617-32c408fa6b8c.mp4


## Testing scenarios

- [x] create new user with the new onboarding team by default
  - create new account
  - check /tasks page
  - see 3 seed tasks, including 1 more `invite` task

- [x] create new team
  - check /team page
  - see 1 seed task, including `invite` task, click to open a new tab, then open the invite modal.

- [x] create new team under new org
  - check /team page
  - see 1 seed task, including `invite` task, click to open a new tab, then open the invite modal.

## Final checklist
